.github/workflows/integration.yaml

```diff
@@ -38,25 +38,22 @@ jobs:
           pip install tox
       - name: Run test
         run: tox -e unit
-        #  Integration Tests won't be executed for now as there are still some
-        #  issues with pytest-operator and k8s charms.
-        #
-        #  integration-test:
-        #    name: Integration tests
-        #    runs-on: ubuntu-latest
-        #    steps:
-        #      - name: Check out code
-        #        uses: actions/checkout@v2
-        #      - name: Setup Python
-        #        uses: actions/setup-python@v2
-        #        with:
-        #          python-version: 3.8
-        #      - name: Install Dependencies
-        #        run: |
-        #          pip install tox
-        #      - name: Setup operator environment
-        #        uses: charmed-kubernetes/actions-operator@master
-        #        with:
-        #            provider: microk8s
-        #      - name: Run test
-        #        run: tox -e integration -- -m container
+  integration-test:
+    name: Integration tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+      - name: Setup Python
```

Comment:
Are we testing 3.8 specifically? If not, we can probably just skip this step and use whatever system Python version is installed

Reply:
I think this pattern started before the `ubuntu-latest` tag was updated back in March.  It now comes with 3.8, so we can certainly try dropping this.